source/MaterialXGenOgsFx/OgsFxShaderGenerator.h

```diff
@@ -24,9 +24,9 @@ class OgsFxShaderGenerator : public GlslShaderGenerator
     /// Return a unique identifyer for the target this generator is for
     const string& getTarget() const override { return TARGET; }
 
-    /// Generate a shader starting from the given element, translating 
+    /// Generate a shader starting from the given element, translating
     /// the element and all dependencies upstream into shader code.
-    ShaderPtr generate(const string& shaderName, ElementPtr element, GenContext& context) const override;
+    ShaderPtr generate(const string& name, ElementPtr element, GenContext& context) const override;
```

Comment:
Trivial: argument name change?

Reply:
Ya, it was supposed to be name, was wrong in the header.